src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFANewModifyPage.java

```diff
@@ -76,65 +60,85 @@ public void setParentAccountTextField(final String parentAccount) {
 
     /**
      * Method to set accountNumber attribute.
+     *
      * @param accountNumber the value.
      */
     public void setAccountNumberTextField(final String accountNumber) {
+        this.accountNumberTextField.clear();
         this.accountNumberTextField.sendKeys(accountNumber);
     }
 
     /**
      * Method to set accountSite attribute.
+     *
      * @param accountSite the value.
      */
     public void setAccountSiteTextField(final String accountSite) {
+        this.accountSiteTextField.clear();
         this.accountSiteTextField.sendKeys(accountSite);
     }
 
     /**
      * Method to set type attribute.
+     *
      * @param type the value.
      */
     public void setAccountTypeList(final String type) {
+        //this.typeList.clear();
         this.typeList.sendKeys(type);
     }
 
     /**
      * Method to set industry attribute.
+     *
      * @param industry the value.
      */
     public void setIndustryList(final String industry) {
+        //this.industryList.clear();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)